preliminary-work/nematic_order.py

```diff
@@ -0,0 +1,41 @@
+import flowermd
+import hoomd
+import gsd
+import numpy as np
+import freud
+import time
+
+
+def nematic_order(gsd_file):
+    # Starting timer to see how long this process takes
+    start = time.process_time()
+    s2_orders = []
+    count = 0
+    traj = gsd.hoomd.open(gsd_file)
+    nematic = freud.order.Nematic()
+    # Iterate through every frame in the trajectory
+    for frame in traj:
```

Comment:
Use the start, stop and stride parameters here when iterating through the frames of the trajectory.